Library/Homebrew/test/utils_spec.rb

```diff
@@ -0,0 +1,293 @@
+require "utils"
+
+describe self do
+  let(:dir) { @dir = Pathname.new(Dir.mktmpdir) }
+
+  after(:each) do
+    dir.rmtree unless @dir.nil?
+  end
```

Comment:
Could this be one line? I think it's better for little things like this because it avoid drawing attention to them.